gc/base/Collector.hpp

```diff
@@ -49,6 +49,11 @@ class MM_Collector : public MM_BaseVirtual
 	/*
 	 * Data members
 	 */
+public:
+	/* The following start/end times record total cycle duration, done only by main thread. */
+	uint64_t _collectionStartTime;
```

Comment:
Could we move these to `Scavenger.hpp`? no other collector will need these, scavenger is a special case for requiring these

Reply:
You can access them with `extensions->_scaveneger` in MemorySubSpaceSemiSpace